content/general-docs/introduction.adoc

```diff
@@ -160,4 +160,9 @@ Notes, footnotes, and examples shall not contain requirements or any information
 
 The following deliverables are provided for {THIS_STANDARD}:
 
-> Define the deliverables of this project here
+* ASAM OpenMATERIAL 3D Specification
```

Comment:
we need to create the link to the standard itself. There are also a couple of name rules to comply:
(see ASM-41(https://internal.pages.asam.net/asam-editorial-guide/editorial-guide-generator/editorialguide-asciidoc/1.0.0/Editorial_guide/01_basics.html#sec-ASM-41) and ASM-137 from Editorial Guide)

It could look like the following:
- ASAM OpenMATERIAL®3D BS 1.0.0 Specification, YYYY-MM-DD (this document, contained in this site)

Reply:
I added it like you suggested.
Locally adding the (R) did not work. I checked and OpenDRIVE also does not have it (not sure though, if ASAM has a trade mark on that.
And I also left the date blank and we need to fill that right before we do the release next year.

Reply:
Is that okay for you @AsamDiegoSanchez ?